### PR TITLE
Fix commit compare style

### LINKED ISSUE #25825: Compare commit page UI problem
### Description

image

You can confirm this issue in try.gitea.io:
https://try.gitea.io/yp05327/testrepo/compare/7287e11b139eeb913348599efd543baf77b00709..73bc61b468580fd399568cd214bef3acf684bad9

### Gitea Version

latest

### Can you reproduce the bug on the Gitea demo site?

Yes

### Log Gist

_No response_

### Screenshots

_No response_

### Git Version

_No response_

### Operating System

_No response_

### How are you running Gitea?

build

### Database

None

### PR DESCRIPTION
**Pull request by puni9869**
as title

Fixes : #25825
Before
image

After change
image

### CODE REVIEW

File: templates/repo/commits_table.tmpl [hunk 1]
```diff
@@ -8,7 +8,7 @@
 			{{.locale.Tr "repo.commits.no_commits" $.BaseBranch $.HeadBranch}} {{if .RefName}}({{.RefName}}){{end}}
 		{{end}}
 	</div>
-	<div class="commits-table-right">
+	<div class="commits-table-right gt-whitespace-nowrap">
```
Comment (delvh) [inline templates/repo/commits_table.tmpl:11]: Could you please add a screenshot?
I have some trouble imagining what the change is.

Comment (delvh) [reply templates/repo/commits_table.tmpl:11]: Not necessary anymore, the linked issue suffices.

**Review by delvh — APPROVED** [review]

File: templates/repo/commits_table.tmpl (hunk 1 above)
Comment (puni9869) [reply templates/repo/commits_table.tmpl:11]: Done, you checked latter  😆

**Comment by puni9869** [comment]
+1 more needed to walk this out.

**Review by yardenshoham — APPROVED** [review]

**Review by silverwind — APPROVED** [review]

**Pull request merged**